csg/src/libcsg/modules/io/lammpsdatareader.cc

```diff
@@ -328,7 +328,8 @@ void LAMMPSDataReader::ReadBox_(std::vector<std::string> fields,
 
     m(i, i) = std::stod(fields.at(1)) - std::stod(fields.at(0));
   }
-  top.setBox(m * tools::conv::ang2nm);
+//  top.setBox(m * tools::conv::ang2nm);
```

Comment:
Why is this needed?

Reply:
> @marvinbernhardt I don't have much experience with `csg_ml` could you have a look at this PR?
> 
> @satyenvd where is this code from and could you add documentation & a tutorial?

I can have a look, but not much expertise when it comes to C++ code quality. Anyway, is there a paper or something to read about the idea behind the code?